PeakLocalisations/Localisation/App Store/Screenshots.xcstrings

```diff
@@ -103,6 +157,12 @@
     "Widgets.Lockscreen" : {
       "extractionState" : "manual",
       "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "...Und für deinen Sperrbildschirm"
```

Comment:
It is a continuation of the previous sentence that still somewhat stands on its own (it's used in screenshots and both texts are used in a separate screenshot where the second follows the first).

<img width="892" alt="image" src="https://github.com/HarshilShah/PeakLocalisations/assets/1689782/28ccd71e-6b04-40ab-b23b-bd8638c972a4">

But your points are valid. Btw. this is the only text that is not using the unicode character in English either, so that's why I kept the three dots.
@HarshilShah is there a specific reason for this here in English or just an oversight?

Reply:
Yeah definitely just an oversight, will fix